picsur/docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: '3.7'
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: picsur_web_1
+      APP_PORT: 8286
+
+  db:
+    image: postgres:16@sha256:cef2d22004db69e3d601442ca4cac142adda0987ad7ca4f28c4e998bef690951

```

I would use a newer version of postgres.
```suggestion
    image: postgres:17.4@sha256:fe3f571d128e8efadcd8b2fde0e2b73ebab6dbec33f6bfe69d98c682c7d8f7bd

```